Apps/Contoso.WPF.Puppet/MainWindow.xaml

```diff
@@ -93,7 +93,21 @@
                     </GroupBox>
                 </StackPanel>
             </TabItem>
-            <TabItem Header="Others"/>
+            <TabItem Header="Others">
+                <StackPanel>
+                    <CheckBox Name="countryCodeEnabledCheckbox" Content="Country Code Enabled" Margin="0,10,0,10" Checked="CountryCodeEnabled_Checked" Unchecked="CountryCodeEnabled_Checked"/>
```

Comment:
There is a warning when you use camelCase. We fixed that everywhere in the apps in previous PRs.

Please fix:

![Capture](https://user-images.githubusercontent.com/623672/60044668-b72d8880-9677-11e9-8397-66dca9aa8c44.PNG)